t/api/15-search.t

```diff
@@ -133,8 +133,11 @@ subtest 'Errors' => sub {
     $t->get_ok('/api/v1/experimental/search?q=*', 'wildcard is interpreted literally');
     $t->json_is('/data' => [], 'no result for *');
 
-    $t->app->config->{rate_limits}->{search} = 3;
-    $t->get_ok('/api/v1/experimental/search?q=timezone', 'Search succesful') for (1 .. 3);
+    $t->app->config->{rate_limits}->{apiv1_search_query} = 3;
+    $t->get_ok('/api/v1/experimental/search?q=timezone', {Accept => 'application/json'})
+      ->content_type_is('application/json;charset=UTF-8')
+      for (1 .. 3);
```

Comment:
I would check `status_is` on all 3 requests, just for that extra little bit of test coverage. 😁